rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all places where community members interact, including Discourse, Matrix, and GitHub
```

Comment:
"All places" is too broad. If me and Graham meet up in the pub is the Community Team expected to be there moderating? Or what if someone opens a ticket on my personal Nix-related project?

How able "Moderate all official community communication channels."

Reply:
thanks, this looks like an oversight on our end. we will probably address it at the same time as https://github.com/NixOS/rfcs/pull/98#discussion_r682173351